index.js

```diff
@@ -17,14 +16,14 @@ require('react-native').unstable_enableLogBox();
 
 enableScreens();
 
-const Elegant = () => (
-  <Provider store={store}>
-    <PersistGate loading={<Loading />} persistor={persistor}>
-      <PaperProvider>
+const Elegant = () => {
+  return (
+    <Provider store={store}>
+      <PersistGate loading={<Loading />} persistor={persistor}>
         <App />
-      </PaperProvider>
-    </PersistGate>
-  </Provider>
-);
+      </PersistGate>
+    </Provider>
+  );
+};
```

Comment:
In the execution context, there isn't anything. so why you do not write like below:

```javascript
const Elegant = () => (
  <Provider store={store}>
    <PersistGate loading={<Loading />} persistor={persistor}>
      <App />
    </PersistGate>
  </Provider>
);
```